.github/workflows/lint.yml

```diff
@@ -17,4 +17,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v7
         with:
-          version: "v1.61.0"
+          version: "v2.0.2"
```

Comment:
I noticed we aren't using the classical `tools` approach in ecdysis, so we could get notified about newer versions. I can add it if we think it's worth it.